crates/red_knot_python_semantic/resources/mdtest/annotations/literal_string.md

```diff
@@ -73,12 +73,12 @@ qux = (foo, bar)
 reveal_type(qux)  # revealed: tuple[Literal["foo"], Literal["bar"]]
 
 # TODO: Infer "LiteralString"
-reveal_type(foo.join(qux))  # revealed: @Todo(Attribute access on `StringLiteral` types)
+reveal_type(foo.join(qux))  # revealed: @Todo(decorated method)
```

Comment:
It doesn't look like it, but this is progress. We now understand attribute access on `StringLiteral` types, but `join` is an overloaded method.